portfolio/src/main/webapp/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA4Nn1hb3fHMxsfw3WdbxYelezg-6RFInQ"></script>
```

Comment:
document why you have async and defer (what effect did you want)?  i assume you dont' want it to block loading the page as you try to connect to the maps (and if it doesn't work, you still load the page.)